lunes_cms/cms/admin.py

```diff
@@ -76,3 +78,4 @@ def get_app_list(self, request):
 admin.site.register(Document, DocumentAdmin)
 admin.site.register(GroupAPIKey, GroupAPIKeyAdmin)
 admin.site.unregister(APIKey)
+admin.site.register(FeedbackAdmin)
```

Comment:
```suggestion
admin.site.register(Feedback, FeedbackAdmin)
```
You need to pass the model to the register function as well.
This requires the model to be imported in line 11 obviously:
```
from .models import Discipline, TrainingSet, Document, GroupAPIKey, Feedback
```